dbt_project.yml

```diff
@@ -13,3 +13,63 @@ target-path: "target"
 clean-targets:
   - "target"
   - "dbt_packages"
+
+vars:
+  snowplow_normalize:
+    # Sources
+    # snowplow__atomic_schema: 'atomic_data_sample' # Only set if not using 'atomic' schema for Snowplow events data
+    # snowplow__database:  # Only set if not using target.database for Snowplow events data -- WILL BE IGNORED FOR DATABRICKS
+    snowplow__events: "{{ source('atomic', 'events') }}"
+    # Variables - Standard Config
+    snowplow__start_date: '2020-01-01'
+    snowplow__backfill_limit_days: 30
+    snowplow__app_id: []
+    snowplow__derived_tstamp_partitioned: true
+    # Variables - Advanced Config
+    snowplow__lookback_window_hours: 6
+    snowplow__days_late_allowed: 3
+    snowplow__upsert_lookback_days: 30
```

Comment:
Yes agreed, I do think keeping the lookback window wouldn't be a bad thing though as it's the easiest way to reprocess some amount of data, but this can be part of the unifying web and mobile, then bring all other packages to the same method